.github/CONTRIBUTING.md

```diff
@@ -1,65 +1,56 @@
-  may then send your changes as a pull request (PR) on GitHub.
-* When submitting changes, you confirm that your code is licensed under the terms of the
-  [GNU General Public License v3](https://www.gnu.org/licenses/gpl-3.0.html).
-* Please test (compile and run) your code before you submit changes! Ideally, provide test feedback in the PR
+* Make changes on a separate branch with a meaningful name, not on the _master_ nor the _dev_ branch. This is commonly known as *feature branch workflow*. You may then send your changes as a pull request (PR) on GitHub.
+* Please test (compile and run) your code before submitting changes! Ideally, provide test feedback in the PR
   description. Untested code will **not** be merged!
-* Try to figure out yourself why builds on our CI fail.
-* Make sure your PR is up-to-date with the rest of the code. Often, a simple click on "Update branch" will do the job,
-  but if not, you are asked to rebase the dev branch manually and resolve the problems on your own. You can find help [on the wiki](https://github.com/TeamNewPipe/NewPipe/wiki/How-to-merge-a-PR). That will make the
+* Make sure your PR is up-to-date with the rest of the code. Often, a simple click on "Update branch" will do the job, but if not, you are asked to rebase the dev branch manually and resolve the problems on your own. You can find help [on the wiki](https://github.com/TeamNewPipe/NewPipe/wiki/How-to-merge-a-PR). That makes the
   maintainers' jobs way easier.
-* Please show intention to maintain your features and code after you contributed it. Unmaintained code is a hassle for
-  the core developers, and just adds work. If you do not intend to maintain features you contributed, please think again
-  about submission, or clearly state that in the description of your PR.
+* Please show intention to maintain your features and code after you contributed it. Unmaintained code is a hassle for core developers. If you do not intend to maintain features you contributed, please rethink your submission, or clearly state that in the PR description.
 * Respond yourselves if someone requests changes or otherwise raises issues about your PRs.
 * Send PR that only cover one specific issue/solution/bug. Do not send PRs that are huge and consists of multiple
   independent solutions.
+* Try to figure out yourself why builds on our CI fail.
 
 ## Communication
 
-* There is an IRC channel on Freenode which is regularly visited by the core team and other developers:
-  [#newpipe](irc:irc.freenode.net/newpipe). [Click here for Webchat](https://webchat.freenode.net/?channels=newpipe)!
-* If you want to get in touch with the core team or one of our other contributors you can send an email to
+* The [#newpipe](irc:irc.freenode.net/newpipe) channel on freenode has the core team and other developers in it. [Click here for webchat](https://webchat.freenode.net/?channels=newpipe)!
+* There is also a Matrix room at #newpipe:matrix.org
```

Comment:
```suggestion
* You can also use an existing/new Matrix account to join the Newpipe channel at #freenode_#newpipe:matrix.org.
```

Reply:
You can also use a~~n existing/new~~ Matrix account to join the Newpipe channel at #freenode_#newpipe:matrix.org.

Reply:
Edited. 👍